### PR TITLE
Regen

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
Part of https://github.com/gtk-rs/gir/pull/820.

cc @EPashkin @sdroege

### CODE REVIEW

File: gtk-sys/src/manual.rs [hunk 1]
```diff
@@ -1,7 +0,0 @@
-// Copyright 2013-2017, The Gtk-rs Project Developers.
-// See the COPYRIGHT file at the top-level directory of this distribution.
-// Licensed under the MIT license, see the LICENSE file or <http://opensource.org/licenses/MIT>
-
-pub mod xlib {
-    pub type Window = i32;
```
Comment (EPashkin) [inline gtk-sys/src/manual.rs]: Why is it removed?

Comment (GuillaumeGomez) [reply gtk-sys/src/manual.rs]: That's... a very good question.

Comment (GuillaumeGomez) [reply gtk-sys/src/manual.rs]: And it's back!

**Comment by GuillaumeGomez** [comment]
Supersed by #148.

**Pull request closed without merging**